README.md

```diff
@@ -32,6 +32,34 @@ To create a new version of the documentation:
 
 ## Contributing to docs
 
+### Sign Off on All Commits
+
+All contributions to the docs need to be signed off.
+
+To sign off when creating a commit, run:
+
+    git commit --signoff
```

Comment:
`git commit -s` is good enough.

Reply:
ok. I'll change it to

`git commit -m "Commit message" -s`

because with only the signoff it had an error about the lack of message